pillar/base/sentry.sls

```diff
@@ -0,0 +1,6 @@
+sentry:
```

Comment:
Feels weird that these are all in base, and that all are in clear-text.

It _appears_ to me that everything here is sufficient for someone to report check-ins on our behalf, including [updating the interval](https://docs.sentry.io/product/crons/getting-started/http/#creating-or-updating-a-monitor-through-a-check-in-optional).

Reply:
this makes it more interesting i missed that you could check in on a non existent monitor!

Reply:
wdyt about [1456a65](https://github.com/python/psf-salt/pull/585/commits/1456a6540f403be2ceff8c11ae9555e96db87b9c)